core/src/main/java/com/exadel/etoolbox/linkinspector/core/servlets/SettingsDataSourceServlet.java

```diff
@@ -0,0 +1,114 @@
+package com.exadel.etoolbox.linkinspector.core.servlets;
+
+/**
```

Comment:
(cosmetic) Please observe this notice is put before the imports section here , but the same one is put _after_ the imports section in the previous class. Need to put them in the same place,

Do we need these notices at all? No objection from my side, but:
- this is not customary for the project. Should all the other people start adding theirs in all the other classes?
- the authorship is clearly seen from GIT history anyway

Reply:
removed